### PR TITLE
Fix #1131 - Adds status, creation and last changed time to issue db

### LINKED ISSUE #1131: Discussion for Issue database needs a migration script.
I have some questions about [Issue #968 - writing a migration script for issues.db backup](https://github.com/webcompat/webcompat.com/issues/968).

Issue #968 is a blocker for [Issue #865 - Backend implementation of detect bugs with similar domain](https://github.com/webcompat/webcompat.com/issues/865)

The subtasks for the for the script as mentioned by @karlcow in [Issue#968](https://github.com/webcompat/webcompat.com/issues/968):
- [ ] Stop webcompat.com
- [ ] Backup the existing DB file.
- [ ] Delete the DB..
- [ ] "Dump to DB": Parse GitHub (pagination involved, probably many HTTP requests)
      Check the HTTP status code for each request
      Grab the pagination link
      Check that the body contains the required information
      Report any issues during the dump to DB.

Currently there are two scripts in the [Issue-parser repoditory](https://github.com/webcompat/issue_parser):
1. [extract_id_title_url.py](https://github.com/webcompat/issue_parser/blob/master/extract_id_title_url.py) - this script reads the set of all `open` issues from [web-bugs](https://api.github.com/repos/webcompat/web-bugs) and creates two files : 
                  a) webcompatdata-bzlike.json
                  b)webcompatdata.csv

Right now these files contain only 254 issues- the number of issues that were open in the web-bugs repository when I ran the extract_id_title_url.py.
1.  [dump_webcompat_to_db.py](https://github.com/webcompat/issue_parser/blob/master/dump_webcompat_to_db.py) - this script adds the contents from webcompatdata-bzlike.json to the webcompat_issues table in the issues.db file

Additionally I am adding few more subtasks:
- [x] Modify dump to db to read issues irrespective of status (Currently only open issues are extracted)
- [ ] Add separate entry for status in issues.db and update it when the issue status changes

**Questions to be discussed**
1. Should the data that is being backup-ed up be webcompat_issues table or issues.db?
2. Can I use some libraries like: 
                 - [rdiff-backup](http://www.nongnu.org/rdiff-backup/rdiff-backup.1.html)
                 - [bup](https://github.com/bup/bup/blob/master/README.md)
   If we make use of these, the backup data may not be saved in the webcompat.com server. We need to investigate this.
3. Right now, I plan to append timestamp to the backup file every time the script is run. We can set up a crone job to take backup periodically. Why should we stop webcompat.com? Wouldn't the issues reported during backup be extracted the next time the backup is taken?
   Is running a crone job defies the purpose of migration script? I am not sure.
4. Should I also include code for creating a local backup copy in the system of the person running the migration script in addition to server?

Any ideas? @miketaylr @magsout @adamopenweb @karlcow

### PR DESCRIPTION
**Pull request by deepthivenkat**
In addition, I also made use of domain search backend implementation code from https://github.com/dshgna/webcompat.com/tree/issue/865-2

### CODE REVIEW

**Comment by karlcow** [comment]
@deepthivenkat For the Pull request, see the syntax we agreed in London.

https://github.com/webcompat/webcompat.com/blob/master/CONTRIBUTING.md#submission-guidelines. Item 9 of the list. 

For example, this could have been: 

`Fix #1131 - Adds status, creation and last changed time to issue db`

Keep the message short to avoid to be cut.

**Comment by karlcow** [comment]
The commit message as the same problem. :)

**Comment by deepthivenkat** [comment]
Thanks @karlcow! made the changes :)

File: webcompat/webhooks/helpers.py [hunk 1]
```diff
@@ -49,7 +50,30 @@ def set_label(label, issue_number):
     api_post('labels', payload, issue_number)
 
 
-def dump_to_db(title, body, issue_number):
+def extract_domain_name(url):
```
Comment (miketaylr) [inline webcompat/webhooks/helpers.py]: We have a similar method here: https://github.com/webcompat/webcompat.com/blob/d40acd13eb7956f34c7e087f8ae3a1439bb19511/webcompat/form.py#L143

I haven't looked closely, but can we consolidate? Is there some case that this one handles that the existing one doesn't?

**Comment by miketaylr** [comment]
(I don't see a review request in this PR yet, so I'm assuming it's not ready for review...)

**Comment by deepthivenkat** [comment]
No mike..it's not complete yet. I am still working on this one

**Comment by miketaylr** [comment]
👍

**Comment by miketaylr** [comment]
Just to create a record here: we can't merge in this commits as is, given that @dshgna authored the majority here. Please cherry-pick her commits as a base, and work on top of those so she's properly attributed as the author. Thanks!

**Pull request closed without merging**